eahub/templates/components/navbar.html

```diff
@@ -1,100 +1,141 @@
+                        </a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="{% url 'account_signup' %}">
+                            <span class="fa-icon"><i class="fa fa-user-plus"></i></span> Sign Up
+                        </a>
+                    </li>
+                {% endif %}
 
-        <li class="nav-item dropdown">
-          <button class="navbar-toggler nav-link d-lg-flex"
-                  type="button"
-                  aria-expanded="false"
-                  data-toggle="dropdown"
-                  aria-haspopup="true"
-                  role="button"
-                  id="navbar-collapse-lg-content"
-          >
-            <span class="navbar-toggler-icon"></span>
-          </button>
-          <div class="dropdown-menu" aria-labelledby="navbar-collapse-lg-content">
-            <a class="dropdown-item" href="https://resources.eahub.org/">
-              <i class="fa fa-book"></i> EA Resources
-            </a>
-            <a class="dropdown-item" href="https://donationswap.eahub.org">
-              <i class="fa fa-exchange-alt"></i> EA Donation Swap
-            </a>
-            <a class="dropdown-item" href="{% url 'about' %}">
-              <i class="fa fa-info"></i> About
-            </a>
-            <a class="dropdown-item" href="https://feedback.eahub.org">
-              <i class="fa fa-comment"></i> Feedback
-            </a>
-            {% if user.is_authenticated %}
-              <a class="dropdown-item" href="{% url 'account_logout' %}?next={% firstof request.path '/' %}">
-                <i class="fa fa-sign-out-alt" id="logout"></i> Log out
-              </a>
-            {% endif %}
-          </div>
-        </li>
-      </ul>
-    </div>
+                <li class="nav-item">
+                    <a class="nav-link" href="{% url 'profiles' %}">
+                        <span class="fa-icon"><i class="fa fa-users"></i></span>
+                        Profiles
+                    </a>
+                </li>
+                <li class="nav-item">
+                    <a class="nav-link" href="{% url 'groups' %}">
+                        <span class="fa-icon"><i class="fa fa-globe"></i></span>
+                        Groups
+                    </a>
+                </li>
 
-  </div>
+            
+                <li class="nav-item d-flex d-lg-none">
+                    <a class="nav-link"
+                        href="https://resources.eahub.org/"
+                    >
+                        <i class="fa fa-book"></i> EA Resources
+                    </a>
+                </li>
+                <li class="nav-item d-flex d-lg-none">
+                    <a class="nav-link"
+                        href="https://donationswap.eahub.org"
+                    >
+                        <i class="fa fa-exchange-alt"></i> EA Donation Swap
```

Comment:
Can we have external link symbols here?

Reply:
done

I haven't invested a lot of time sadly, hence it looks a bit ugly (two icons on one line is somehow harder to make professional)

![image](https://user-images.githubusercontent.com/36416236/106384496-165e8d80-63c3-11eb-9acc-4413a6a795d1.png)